docs/docs-content/automation/crossplane/deploy-cluster-aws-crossplane.md

```diff
@@ -0,0 +1,653 @@
+              config structure\n      # See docs/additional-daemonsets.md for more information\n      #\n      #
+              example:\n      #   nodeSelector:\n      #     node.kubernetes.io/instance-type:
+              c5.large\n      #   volumeAttachLimit: 15\n\n    # Enable compatibility for the A1 instance family via use
+              of an AL2-based image in a separate DaemonSet\n    # a1CompatibilityDaemonSet:
+              true\n\n    #storageClasses: []\n    # Add StorageClass resources like:\n    # - name: ebs-sc\n    #   #
+              annotation metadata\n    #   annotations:\n    #     storageclass.kubernetes.io/is-default-class:
+              \"true\"\n    #   # label metadata\n    #   labels:\n    #     my-label-is: supercool\n    #   # defaults
+              to WaitForFirstConsumer\n    #   volumeBindingMode: WaitForFirstConsumer\n    #   # defaults to
+              Delete\n    #   reclaimPolicy: Retain\n    #   parameters:\n    #     encrypted:
+              \"true\"\n\n    volumeSnapshotClasses: []\n    # Add VolumeSnapshotClass resources like:\n    # - name:
+              ebs-vsc\n    #   # annotation
+              metadata\n    #   annotations:\n    #     snapshot.storage.kubernetes.io/is-default-class:
+              \"true\"\n    #   # label metadata\n    #   labels:\n    #     my-label-is: supercool\n    #   #
+              deletionPolicy must be specified\n    #   deletionPolicy: Delete\n    #   parameters:\n\n    # Use old
+              CSIDriver without an fsGroupPolicy set\n    # Intended for use with older clusters that cannot easily
+              replace the CSIDriver object\n    # This parameter should always be false for new
+              installations\n    useOldCSIDriver: false"
+
+      providerConfigRef:
+        name: default
+    ```
+
+    The cluster profile contains the following core infrastructure layers.
+
+    | **Pack Type** | **Registry** | **Pack Name** | **Pack Version** |
+    | ------------- | ------------ | ------------- | ---------------- |
+    | OS            | Public Repo  | `ubuntu-aws`  | `22.04`          |
+    | Kubernetes    | Public Repo  | `kubernetes`  | `1.29.0`         |
+    | Network       | Public Repo  | `cni-calico`  | `3.27.0`         |
+    | Storage       | Public Repo  | `csi-aws-ebs` | `1.26.1`         |
+
+    :::tip
+
+    You can access the Palette UI and simulate creating a cluster profile to gather the packs' required values to create
+    a cluster profile through Crossplane. During the cluster profile creation, click the API button to display the API
+    payload. You will find the values of each pack's `name`, `tag`, `uid`, `registryUid`, and `values` parameters.
+
+    :::
+
+16. Create the cluster profile.
+
+    ```bash
+    kubectl apply --filename crossplane-aws/cluster-profile-aws.yaml
+    ```
+
+17. Issue the commands below to get the ID of the cluster profile once its creation is complete.
```

Comment:
Let's include the output as a separate code snippet below :)

Reply:
:white_check_mark: